doc/content/microreactors/gcmr/GCMR_Core_Neutronics.md

```diff
@@ -109,6 +101,10 @@ The first step is to generate homogenized multi-group cross-sections using Serpe
 
 
```

Comment:
let s create a subsection and give some text (notably telling people they need to use LFS, and what version of Serpent)

this is nested under ## Griffin model, does not really make sense